virttest/utils_libvirt/libvirt_memory.py

```diff
@@ -79,6 +80,8 @@ def consume_vm_freememory(vm_session, consume_value=100000):
     """
     vm_session.cmd_status("swapoff -a")
     free_mem = utils_memory.freememtotal(vm_session)
+    if not utils_package.package_install("numactl", vm_session):
+        raise exceptions.TestError("Fail to install memhog")
```

Comment:
Seems we could make the error message a little bit more clearer by adding `numactl` into it. Something like 
`"Fail to install pkg numactl which provide command 'memhog'"`

Reply:
Updated. thanks